src/services/utilities.ts

```diff
@@ -758,7 +758,7 @@ namespace ts {
         if (isClassDeclaration(node)) {
             // for class and function declarations, use the `default` modifier
             // when the declaration is unnamed.
-            const defaultModifier = find(node.modifiers!, isDefaultModifier);
+            const defaultModifier = node.modifiers && find(node.modifiers, isDefaultModifier);

```

Comment:
Could also do

```suggestion
            const defaultModifier = forEach(node.modifiers, isDefaultModifier);
```

or

```suggestion
            const defaultModifier = node.modifiers?.find(isDefaultModifier);
```